packages/node_modules/@webex/widget-recents/src/components/RecentsHeader.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import {Button, Icon, SearchInput} from '@momentum-ui/react';
+import {Button, Icon, InputSearch} from '@momentum-ui/react';
```

Comment:
It got changed from `SearchInput` to `InputSearch` (https://momentum.design/components/input-search/code#default) 

Working recent widgets after the change:

<img width="700" alt="Screenshot 2020-07-09 at 4 41 10 AM" src="https://user-images.githubusercontent.com/55378170/86979289-88108900-c19e-11ea-90b0-098c082ad91b.png">

Reply:
Thanks for addressing the breaking change